### ISSUE TITLE
rename(CUT=X.cut, COLOR='color') - counterintuitive IMO

### ISSUE THREAD

**Issue by simjbaum** [kieferk/dfply#101]
Hi,
I found the orientation of renaming the columns counter intuitive in rename(CUT=X.cut, COLOR='color')

"column name wanted" =  "old column name" 
"old column name"        = "column name wanted" 

best
Simon

**Comment by norabelrose** [comment]
I agree with this too actually, but it mirrors the usage in the dplyr R package. Including a different function with the reverse argument orientation could be nice though.
